latex/epoch.tex

```diff
@@ -146,11 +269,11 @@ \subsection{Stake Distribution Calculation}
       \nextdef
       & \fun{rewardStake} \in (\AddrRWD\mapsto\Coin) \to \Stake \\
       & \fun{rewardStake}~{rewards} =
-          \{\fun{stakeHK_r}~{addr}\mapsto c \mid a\mapsto c\in\var{rewards}\} \\
+          \{\fun{stakeHK_r}~{a}\mapsto c \mid a\mapsto c\in\var{rewards}\} \\
       \nextdef
       & \fun{poolStake} \in \HashKey \to \powerset{\HashKey} \to (\HashKey \mapsto \HashKey)
           \to \Stake \to \Stake \\
-      & \poolStake{operator}{owners}{stake}{delegations} = \\
+      & \poolStake{operator}{owners}{delegations}{stake} = \\
```

Comment:
Are we not counting the stake that owners contribute to the size of the stake pool?

Reply:
Maybe I'm not understanding the issue. The set on the rhs of the union maps the operator's hashkey to the sum total of all stake owned by the stake pool operator and all owners. that way, from this point onward, we only have to check the pool operator's hashkey to see if the pledge has been met, calculate the leader rewards, etc.